package.json

```diff
@@ -105,17 +105,18 @@
     "@pollyjs/core": "^6.0.5",
     "@pollyjs/node-server": "^6.0.1",
     "@pollyjs/persister-fs": "^6.0.5",
+    "@testing-library/react": "^12.1.2",
```

Comment:
This line will be removed once we have resolved the master branch commit issue.

Reply:
Do we need to add testing-library in this PR, since we are going to be working on task to add that separately.

Reply:
I have removed it and rebased the PR.